odk1-src/collect-form-map.rst

```diff
@@ -29,7 +29,7 @@ Filled forms are represented by colored map markers. The color of each map marke
 * Green: sent
 * Red: send attempt failed
 
-Tapping on a map marker will open the filled form for viewing or editing depending on the form status. By default, forms with :formstate:`Saved`, :formstate:`Finalized`, and :formstate:`Submission Failure` status are opened for edit and :formstate:`Sent` forms are opened for viewing. However, if the :guilabel:`Edit Saved Form` button is made unavailable from the :ref:`admin settings <user-access-control-settings>`, forms with :formstate:`Saved`, :formstate:`Finalized`, and :formstate:`Submission Failure` status are opened as view-only.
+Tapping on a map marker will show details about the filled form at the bottom of the screen. There will also be a button for viewing or editing depending on the form status. By default, forms with :formstate:`Saved`, :formstate:`Finalized`, and :formstate:`Submission Failure` status are opened for edit and :formstate:`Sent` forms are opened for viewing. However, if the :guilabel:`Edit Saved Form` button is made unavailable from the :ref:`admin settings <user-access-control-settings>`, forms with :formstate:`Saved`, :formstate:`Finalized`, and :formstate:`Submission Failure` status are opened as view-only.
```

Comment:
I think it would be good to have a screenshot here just to demonstrate the feature. It feels strange to have to read through and picture what the user will see.